src/calendar/tests/CalendarHighlight.spec.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react/addons';
+import CalendarHighlight from '../CalendarHighlight';
+
+const TestUtils = React.addons.TestUtils;
+
+describe('The CalendarHighlight Component', function () {
+  beforeEach(function () {
+    this.spyCx = spyOn(CalendarHighlight.prototype.__reactAutoBindMap, 'cx').and.returnValue('my-class');
```

Comment:
That sounds good to me, would love for you to go ahead with that so we can get this merged in! :)